care/emr/reports/report_type_utils.py

```
@@ -0,0 +1,9 @@
+from django.db import models
+
+
+def validate_associating_id(
+    associating_model: type[models.Model],
+    associating_id: str,
+    report_type_key: str,
```

Ruff (0.14.6)

Unused function argument: `report_type_key`

(ARG001)